task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,9 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    return num % 3 === 0 
+    ? num % 5 === 0 ? 'FizzBuzz' : 'Fizz'
+    : num % 5 === 0 ? 'Buzz' : num
```

Comment:
Please, avoid this code style, if-else is more readable than multiple conditional operators ? :